daemon/create_unix.go

```diff
@@ -42,7 +45,19 @@ func createContainerPlatformSpecificSettings(container *Container, config *runco
 			return fmt.Errorf("cannot mount volume over existing file, file exists %s", path)
 		}
 
-		v, err := createVolume(name, config.VolumeDriver)
+		volumeDriver := config.VolumeDriver
+		if destination != "" && img != nil {
+			if _, ok := img.ContainerConfig.Volumes[destination]; ok {
+				logrus.Errorf("container1: v %s", img.ContainerConfig.Volumes[destination])
```

Comment:
is this a debug statement? is it necessary? the message is very misleading. Can we either use `Debugf` and a better message, or remove the log all together?

(Same three lines below :smile_cat: )